docker/package/scripts/udev-rules

```diff
@@ -9,7 +9,7 @@
 # Don't add udev rules in case the package is installed inside either docker or podman container.
 # Otherwise, post-installation script will fail due to inability to non-zero exit code of the
 # 'udevadm control --reload-rules' call
-if [ ! -f /.dockerenv ] && [ ! -f /.containerenv ] ; then
+if [ ! -f /.dockerenv ] && [ ! -f /.containerenv ] && [ -z "${KUBERNETES_SERVICE_HOST++}" ] ; then
```

Comment:
It seems that we should also update the comment above and mention k8s

Reply:
Doesn't it still fits the description? AFAIU k8s uses docker

Reply:
Yes, but IMO it won't hurt much to mention why we still check for `KUBERNETES_SERVICE_HOST` env variable (since there is neither `.dockerenv` nor `.containerenv`) just not to forget about it over time